opal/core/pathway/pathways.py

```diff
@@ -21,14 +22,57 @@ def redirect_url(self, user=None, patient=None, episode=None):
 
 
 class Pathway(discoverable.DiscoverableFeature):
-    module_name = "pathways"
-    pathway_service = "Pathway"
+    module_name        = "pathways"
+    pathway_service    = "Pathway"
     finish_button_text = "Save"
     finish_button_icon = "fa fa-save"
+    icon               = None
+    display_name       = None
 
     # any iterable will do, this should be overridden
     steps = []
 
+    @classmethod
+    def get_slug(klass):
+        """
+        Returns a string which should be used as the slug for this pathway
+        """
+        slugattr = getattr(klass, 'slug', None)
+        if slugattr:
+            if isinstance(slugattr, string_types):
+                return slugattr
+        return slugify(klass.__name__)
+
+    @classmethod
+    def get_absolute_url(klass, **kwargs):
+        """
+        Returns a string which is the absolute URL of this Pathway.
+        """
+        return '{0}#/{1}/'.format(reverse('pathway_index'), klass.get_slug())
```

Comment:
so by default we're saying that a pathway will not receive a patient/episode which I think makes sense.

We have to have one default, and this is the default we would use in a menu item. I think this is correct.